locales/it/index.yml

```diff
@@ -2717,7 +2717,7 @@ bonus:
         description: Descrizione
         contactInfo: Contatti e informazioni
       cta:
-        website: Vai al sito del partner
+        website: Visita al sito del partner
```

Comment:
Update the string to: Visita il sito del partner

Reply:
OMG sucha typo